pkg/api/policy/v1alpha1/type.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
TODO: revert this

Reply:
rename to `pkg/apis`

Reply:
lint should ignore `pkg/apis`, `pkg/client-go`

Reply:
completed